### ISSUE TITLE
Sword Cross Outline

### ISSUE THREAD

**Issue by KrisLau** [Templarian/MaterialDesign#7235]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://pictogrammers.com/library/mdi) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](https://pictogrammers.com/docs/library/mdi/guides/brand-icons/) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

So currently I'm using the [solid version](https://pictogrammers.com/library/mdi/icon/sword-cross/) to represent an opposing team since there don't seem to be any other icons that fit the bill. It would be nice to have a simple outline version that would be able to use it as a button icon as well though. Currently the solid version doesn't fit with the other icons i use because they are all outlines haha

### Give us an example of what this icon might look like.

Exactly like the solid one but outlined
https://pictogrammers.com/library/mdi/icon/sword-cross/

**Comment by MrGrigri** [comment]
I do not believe that there is enough space to have an outlined version of the crossed swords.

**Comment by masi** [comment]
How about an alternate version that has the sword with the handle on the left on top. Additionally the sword on top has a fine line at the center of the blade.
